packages/server/src/database.ts

```diff
@@ -1,16 +1,45 @@
+import { badRequest, created, OperationOutcomeError, parseSearchRequest, sleep } from '@medplum/core';
+import { AsyncJob } from '@medplum/fhirtypes';
+import assert from 'node:assert';
+import { readFileSync } from 'node:fs';
+import { resolve } from 'node:path';
 import { Pool, PoolClient } from 'pg';
+import * as semver from 'semver';
 import { MedplumDatabaseConfig, MedplumServerConfig } from './config';
+import { getSystemRepo } from './fhir/repo';
 import { globalLogger } from './logger';
+import * as dataMigrations from './migrations/data';
 import * as migrations from './migrations/schema';
-import { sleep } from '@medplum/core';
+import { getServerVersion } from './util/version';
 
 export enum DatabaseMode {
   READER = 'reader',
   WRITER = 'writer',
 }
 
+const UNKNOWN = -1;
```

Comment:
Minor nit:  Let's either give these a prefix (i.e., `DATA_VERSION_UNKNOWN`) or wrap them in a const object